docs/source/cuda/cuda_array_interface.rst

```diff
@@ -63,6 +63,12 @@ The following are optional entries:
     This is for describing more complicated types.  This follows the same
     specification as in the `numpy array interface`_.
 
+- **mask**:  ``None`` or `` object with __cuda_array_interface__``
+
+    All elements of the mask array should be interpreted only as true or not true
+    indicating which elements of this array are valid.The shape of this object
```

Comment:
Need spaces after ".".    Can you also note in the docs that Numba does not currently support working with masked CUDA arrays, and will raise an exception if one is passed to a kernel?

Reply:
Should be resolved now with commit . 
https://github.com/numba/numba/pull/4192/commits/955fad1de446bb24860ad3a7445f146444c26cee